phpBB/phpbb/db/migration/data/v33x/bot_update_vol2.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v33x;
+
+class bot_update_vol2 extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
```

Comment:
```suggestion
	public static function depends_on()
```

Reply:
Shall I adjust this in the old bot update script as well?